src/components/step/step.css

```diff
@@ -0,0 +1,54 @@
+html {
+    font-size: 17px;
+    background-color: #FFFFFF;
+}
+
+@supports (font: -apple-system-body) {
+    html {
+        font: -apple-system-body;
+    }
+}
+
+body,div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,h6,pre,form,fieldset,input,textarea,p,blockquote,th,td { 
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Helvetica, Arial, sans-serif;
+    display: block !important;
+    background-color: #FFFFFF;
+}
+
+a {
+    color: #5381c3;
+    text-decoration: none;
+}
+
+.step-container {
+    padding: 10px 20px 10px 20px;
```

Comment:
So I'd probably remove the top padding here and instead have it come from a top margin on the step title.  In particular that would enable the video to be fully anchored to the top of the screen if there is no title/text.

Reply:
(Removing the left/right padding here and instead adding it to the various child elements I think i'd also be slightly in favor of, so you don't have to do extra stuff to make the YoutubeStep "escape" the frame - might be more stuff like that in the future)